sdcm/sct_events/database.py

```diff
@@ -115,8 +115,9 @@ def add_info(self: T_log_event, node, line: str, line_number: int) -> T_log_even
                                    regex="compaction_stopped_exception")
 DatabaseLogEvent.add_subevent_type("BAD_ALLOC", severity=Severity.ERROR,
                                    regex="std::bad_alloc")
+# Due to scylla issue https://github.com/scylladb/scylladb/issues/19093, we need to suppress the below error
 DatabaseLogEvent.add_subevent_type("SCHEMA_FAILURE", severity=Severity.ERROR,
-                                   regex="Failed to load schema version")
+                                   regex=r'(.*ERROR|!ERR).*Failed to load schema version')
```

Comment:
in `DATABASE_ERROR` we use `(^ERROR|!\s*?ERR).*` for matching errors, possibly we should use the same.
I'm not sure of the log format we get here -  I see in unit tests multiple formats.

Reply:
The unit test failed when I used "^", I wasn't sure how updated our system.log is.